docker/docker-compose.dev.yml

```diff
@@ -55,6 +55,7 @@ services:
     image: immich-web-dev:latest
     build:
       context: ../web
+      target: dev
```

Comment:
the web Dockerfile is only ever used for development so you can revert related changes.

Reply:
Sorry for taking so long. I have removed the `target: dev` and removed the `COPY` from the `web/Dockerfile` completely so that it can only be used with the source volume.